specs/electra/beacon-chain.md

```diff
@@ -363,9 +381,7 @@ class ExecutionPayloadHeader(Container):
     withdrawals_root: Root
     blob_gas_used: uint64
     excess_blob_gas: uint64
-    deposit_requests_root: Root  # [New in Electra:EIP6110]
-    withdrawal_requests_root: Root  # [New in Electra:EIP7002:EIP7251]
-    consolidation_requests_root: Root  # [New in Electra:EIP7251]
+    requests_root: Root  # [New in Electra:EIP6110:EIP7002:EIP7251]
```

Comment:
I believe we don’t need this field at all, as validator requests are already linked to the `BeaconBlockHeader`